02-starting-with-data.Rmd

```diff
@@ -421,25 +414,22 @@ head(sex)
 ```
 
 
-### Using `stringsAsFactors=FALSE`
+### Using `stringsAsFactors = FALSE`
 
 By default, when building or importing a data frame, the columns that contain
-characters (i.e., text) are coerced (=converted) into the `factor` data
-type. Depending on what you want to do with the data, you may want to keep these
+characters (i.e. text) are coerced (= converted) into factors. Depending on what you want to do with the data, you may want to keep these
 columns as `character`. To do so, `read.csv()` and `read.table()` have an
 argument called `stringsAsFactors` which can be set to `FALSE`.
 
-In most cases, it's preferable to set `stringsAsFactors = FALSE` when importing
-your data, and converting as a factor only the columns that require this data
+In most cases, it is preferable to set `stringsAsFactors = FALSE` when importing
+data and to convert as a factor only the columns that require this data
 type.
 
 Compare the output of `str(surveys)` when setting `stringsAsFactors = TRUE`
 (default) and `stringsAsFactors = FALSE`:
 
 
 ```{r, eval=FALSE, purl=FALSE}
```

Comment:
There is a little issue with the formatting here. Could you please delete the "Compare the output..." sentence from the text, bring back the sentences deleted from the chunk, and switch to `purl=TRUE`? this way the code handout will include this chunk.

Reply:
Oh, I see. I was trying to cut the repetition, but I cut it in the wrong place because I didn't know about the handout...